WordPress/src/main/java/org/wordpress/android/ui/reader/services/discover/ReaderDiscoverLogic.kt

```diff
@@ -212,6 +212,10 @@ class ReaderDiscoverLogic @Inject constructor(
                     }
                 }
                 JSON_CARD_INTERESTS_YOU_MAY_LIKE -> {
+                    // We should not have an interests/tags card as the first element on Discover feed.
+                    if (i == 0 && discoverTasks == REQUEST_FIRST_PAGE) {
```

Comment:
❓ Is there a way of checking if the item at index 0 is indeed a tag recommendation card? I am aware that currently that's always true, but just in case the response payload changes (for instance, the 1st tag recommendation card is removed from that initial position) it would be nice to avoid hiding an unwanted card.

Reply:
Maybe I misinterpreted  the code, but it seemed to me that this branch of the `when` already guarantees this is a tag recommendation card. Like, line `214`:

<img width="704" alt="image" src="https://github.com/wordpress-mobile/WordPress-Android/assets/14964993/71f2cbd5-3102-457a-a713-743e147c7622">

If the payload changes and this is no longer the first element, it was my understanding that the first element wouldn't enter this branch.

Reply:
Ah, you're correct, I didn't notice this was all inside a `when` branch that only runs for the tag recommendation card, my bad!

In that case I think the code looks good, thanks!